Fails to compile with v8 3.21.18.2

Starting at commit 619231913c1c2199be64a216e3bebc2a0936efda, I get the following errors when compiling against v8 3.21.18.2 (the latest 3.21 version of v8):

```
/bin/sh /home/cananian/Wikimedia/scribunto-js/v8js/libtool --mode=compile g++ -std=c++11 -I. -I/home/cananian/Wikimedia/scribunto-js/v8js -DPHP_ATOM_INC -I/home/cananian/Wikimedia/scribunto-js/v8js/include -I/home/cananian/Wikimedia/scribunto-js/v8js/main -I/home/cananian/Wikimedia/scribunto-js/v8js -I/usr/include/php5 -I/usr/include/php5/main -I/usr/include/php5/TSRM -I/usr/include/php5/Zend -I/usr/include/php5/ext -I/usr/include/php5/ext/date/lib -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -I/home/cananian/Wikimedia/scribunto-js/v8js/v8/include  -DHAVE_CONFIG_H  -g -O2   -c /home/cananian/Wikimedia/scribunto-js/v8js/v8js_convert.cc -o v8js_convert.lo 
libtool: compile:  g++ -std=c++11 -I. -I/home/cananian/Wikimedia/scribunto-js/v8js -DPHP_ATOM_INC -I/home/cananian/Wikimedia/scribunto-js/v8js/include -I/home/cananian/Wikimedia/scribunto-js/v8js/main -I/home/cananian/Wikimedia/scribunto-js/v8js -I/usr/include/php5 -I/usr/include/php5/main -I/usr/include/php5/TSRM -I/usr/include/php5/Zend -I/usr/include/php5/ext -I/usr/include/php5/ext/date/lib -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -I/home/cananian/Wikimedia/scribunto-js/v8js/v8/include -DHAVE_CONFIG_H -g -O2 -c /home/cananian/Wikimedia/scribunto-js/v8js/v8js_convert.cc  -fPIC -DPIC -o .libs/v8js_convert.o
/home/cananian/Wikimedia/scribunto-js/v8js/v8js_convert.cc:37:13: error: expected unqualified-id before 'static'
   V8_INLINE(static void Copy(const Persistent<S, M>& source,
             ^
/home/cananian/Wikimedia/scribunto-js/v8js/v8js_convert.cc:37:13: error: expected ')' before 'static'
In file included from /home/cananian/Wikimedia/scribunto-js/v8js/php_v8js_macros.h:24:0,
                 from /home/cananian/Wikimedia/scribunto-js/v8js/v8js_convert.cc:27:
/home/cananian/Wikimedia/scribunto-js/v8js/v8/include/v8.h: In instantiation of 'void v8::Persistent<T, M>::Copy(const v8::Persistent<S, M>&) [with S = v8::FunctionTemplate; M2 = v8::CopyablePersistentTraits<v8::FunctionTemplate>; T = v8::FunctionTemplate; M = v8::CopyablePersistentTraits<v8::FunctionTemplate>]':
/home/cananian/Wikimedia/scribunto-js/v8js/v8/include/v8.h:536:14:   required from 'v8::Persistent<T, M>& v8::Persistent<T, M>::operator=(const v8::Persistent<T, M>&) [with T = v8::FunctionTemplate; M = v8::CopyablePersistentTraits<v8::FunctionTemplate>]'
/home/cananian/Wikimedia/scribunto-js/v8js/v8js_convert.cc:421:44:   required from here
/home/cananian/Wikimedia/scribunto-js/v8js/v8/include/v8.h:5572:21: error: 'Copy' is not a member of 'v8::CopyablePersistentTraits<v8::FunctionTemplate>'
   M::Copy(that, this);
                     ^
make: *** [v8js_convert.lo] Error 1
```

We currently test against 3.21.12, so this _should_ work...

--- a/v8js_convert.cc
+++ b/v8js_convert.cc
@@ -37,8 +37,14 @@ namespace v8 {
 		typedef Persistent<T, CopyablePersistentTraits<T> > CopyablePersistent;
 		static const bool kResetInDestructor = true;
 		template<class S, class M>
+#if PHP_V8_API_VERSION >= 3021015
+		static V8_INLINE void Copy(const Persistent<S, M>& source,
+								   CopyablePersistent* dest)
+#else
 		V8_INLINE(static void Copy(const Persistent<S, M>& source,
-								   CopyablePersistent* dest)) {
+								   CopyablePersistent* dest))
+#endif
+		{
 			// do nothing, just allow copy
 		}
 	};

